### DISCUSSION TITLE
shadowsocks over shadow-tls

### DISCUSSION THREAD

**Discussion by love4taylor** [immortalwrt/homeproxy#107] [Q&A]
目前看来没法设置 detour？

**Comment by d4rkblue** [comment]
同需求

**Reply by love4taylor** [reply]
用自定义路由吧，路由节点里添加 stls 和 ss ，然后 ss 设置出站为 stls（也不知道是不是有意不在傻瓜模式支持这种用法

**Reply by 1715173329** [reply]
> 也不知道是不是有意不在傻瓜模式支持这种用法

按目前的架构很难实现，索性不支持了。
说到底用 uci 来管理配置还是过于麻烦。
